packages/admin/cms-admin/src/blocks/EmailLinkBlock.tsx

```diff
@@ -0,0 +1,80 @@
+import { Field, FinalFormInput } from "@comet/admin";
+import { BlockCategory, BlockInterface, BlocksFinalForm, createBlockSkeleton, SelectPreviewComponent } from "@comet/blocks-admin";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface EmailLinkBlockData {
+    email?: string;
+}
+
+interface EmailLinkBlockInput {
+    email?: string;
+}
+
+export const EmailLinkBlock: BlockInterface<EmailLinkBlockData, EmailLinkBlockData, EmailLinkBlockInput> = {
+    ...createBlockSkeleton(),
+
+    name: "Email",
+
+    displayName: <FormattedMessage id="comet.blocks.link.email" defaultMessage="Email" />,
+
+    defaultValues: () => ({ email: undefined }),
+
+    category: BlockCategory.Navigation,
+
+    input2State: (state) => {
+        return state;
+    },
+
+    state2Output: (state) => {
+        return {
+            email: state.email,
+        };
+    },
+
+    output2State: async (output) => {
+        return {
+            email: output.email,
+        };
+    },
```

Comment:
I don't think it's necessary to add these methods, that's what createBlockSkeleton does